_sources/Files/ReadingaFile.rst

```diff
@@ -0,0 +1,82 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: files-3-
+   :start: 1
+
+Reading a File
+~~~~~~~~~~~~~~
+
+As an example, suppose we have a text file called ``qbdata.txt`` that contains
+the following data representing statistics about NFL quarterbacks. Although it
+would be possible to consider entering this data by hand each time it is used,
+you can imagine that it would be time-consuming and error-prone to do this. In
+addition, it is likely that there could be data from more quarterbacks and
+other years. The format of the data file is as follows
+
+::
```

Comment:
We should definitely go with a different example.  This is really outdated and not very inclusive.

We can look at Kaggle and find a subset of something that is much better.